### PR TITLE
AZ-628 - rust toolchain, single point of configuration

### PR DESCRIPTION
**Pull request by lesniak43**

### CODE REVIEW

**Review by fbielejec — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -14,13 +14,16 @@ Aleph node is based on a Substrate node where the default finality gadget (GRAND
 
 ### Installation
 
-1. Install the Substrate framework as described [here][substrate-installation], using the `nightly-2021-10-24` version of the rust toolchain (see the instruction at the bottom of the page)
-2. Clone this repository to your machine
-3. Run `cargo build --release -p aleph-node`
+1. Install `rustup` as described [here][rustup-instalation].
+2. Clone this repository to your machine and go to the top-level directory.
+3. This project is tied to a specific version of the rust toolchain defined in the `rust-toolchain` file. All `rustup` commands, including `cargo`, will respect this setting unless you tell them to do otherwise, so make sure that you have **not** set any [overrides][rustup-overrides] with higher priority.
+4. Run `rustup show` to install the toolchain.
```
Comment (maciejnems) [inline README.md:20]: Wait, `rustup show` installs something? Doesn't it only list the version of rust you have installed?

File: local-tests/send-runtime/rust-toolchain [hunk 2]
```diff
@@ -1 +0,0 @@
-nightly-2021-10-24
```
Comment (maciejnems) [inline local-tests/send-runtime/rust-toolchain:1]: Are we removing it as nightly is not needed for send-runtime and fork-off?

File: README.md (hunk 1 above)
Comment (lesniak43) [reply README.md:20]: There's a four-year-old discussion about this behaviour with no conclusions so far: https://github.com/rust-lang/rustup/issues/1397

File: local-tests/send-runtime/rust-toolchain (hunk 2 above)
Comment (lesniak43) [reply local-tests/send-runtime/rust-toolchain:1]: All rustup tools will walk up the directory tree in search for the rust-toolchain file, as stated in the very first section of https://rust-lang.github.io/rustup/overrides.html
My point is to leave only a single configuration file that will specify the toolchain version.

File: README.md (hunk 1 above)
Comment (maciejnems) [reply README.md:20]: Oh wow :face_with_spiral_eyes:

File: local-tests/send-runtime/rust-toolchain (hunk 2 above)
Comment (maciejnems) [reply local-tests/send-runtime/rust-toolchain:1]: Oh, cool stuff

**Review by maciejnems — APPROVED** [review]

**Pull request merged**